README.md

```diff
@@ -47,9 +67,15 @@ from earthkit.climate.utils import conversions
 pr = precipitation.simple_daily_intensity(precip_data, freq="monthly")
 ```
 
+______________________________________________________________________
+
+## Documentation
+
 For full documentation, including API reference and example notebooks, visit the
 [earthkit-climate ReadTheDocs page](https://earthkit-climate.readthedocs.io)
 
+______________________________________________________________________
+
 ## Development & Contribution Workflow
 
 ### 1. Setup environment (with Pixi)
```

Comment:
OK for this PR but we'll need to re-evaluate the use of pixi in the next WP. This is not used in the rest of the ECMWF stack. Currently, people are moving more towards the use of uv and I believe we should remain consistent with the rest of the stack.